.github/check_version.py

```
@@ -24,31 +24,70 @@
+
+# 2. Fallback to /releases if needed
+if not found_valid_version:
+  print("Falling back to iterating /releases...")
+  resp = requests.get(
+    f"https://api.github.com/repos/{REPO}/releases",
+    headers=headers,
+    timeout=10,
+  )
+
+  if resp.status_code == 404:
+    # No releases yet – treat as version 0.0.0
```

Ruff (0.14.6)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)